services/src/main/java/org/keycloak/services/resources/admin/permissions/RealmPermissionEvaluator.java

```diff
@@ -62,4 +62,8 @@
     void requireViewRequiredActions();
 
     void requireViewAuthenticationFlows();
+
+    void requireViewPerClientConfigDescription();
+
+    void requireViewClientAuthenticatorProviders();
```

Comment:
I'm not sure if you need both methods but just a single one ? I would keep `requireViewClientAuthenticatorProviders`.